### DISCUSSION TITLE
Welcome to mini-typescript Discussions!

### DISCUSSION THREAD

**Discussion by sandersn** [sandersn/mini-typescript#12] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Help each other with exercises and other extensions of mini-typescript.
  * Share ideas.
  * Welcome others and are open-minded. Remember that this is a community we build together.

To get started, comment below with an introduction of yourself and tell us about what you hope to learn here.

**Comment by YaseenYo** [comment]
I am not able to figure out how to try the exercises... @sandersn can you help with this...

**Reply by sandersn** [reply]
Make sure you can build the repo as-is, then add a test for the exercise. For example, to test empty statements, make a new test with source something like `1;;;1`

If you need help with a specific exercise, open a new discussion thread about it.
